dpnp/backend/backend_iface_fptr.hpp

```diff
@@ -59,7 +59,8 @@
  */
 enum class DPNPFuncName : size_t
 {
-    DPNP_FN_NONE,      /**< Very first element of the enumeration */
+    DPNP_FN_NONE,     /**< Very first element of the enumeration */
+    DPNP_FN_ABSOLUTE,      /**< Used in numpy.absolute() implementation  */
```

Comment:
@Rubtsowa These rows show that codestyle was not applied. Please use codestyle tools.